scaling_performance/install_practices.adoc

```diff
@@ -0,0 +1,80 @@
+[[scaling-performance-install-best-practices]]
+= Recommended Installation Practices
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-preinstalling-dependencies]]
+== Pre-installing Dependencies
+
+Save time by ensuring that the RPMs for `atomic-openshift-*`, `iptables`,
+`docker`, and other required packages have been set up before installing
+{product-title}. 
+
+If the dependencies have not been pre-installed, a node host will access the
```

Comment:
On second thought, they're about different things, so I might not link to them. I'll have a better look at the the pre-deployed stuff though. Maybe there's some overlap where there shouldn't be.